patterns/explicit-construction.md

```diff
@@ -0,0 +1,46 @@
+##The Explicit Construction Anti-Pattern
+
+This is the most common anti-pattern. It is easy to fall into this when you haven't really grasped
```

Comment:
I actually did collect real data at a point, but for example the question about it in StackOverflow is referenced by over 300 questions in stack overflow http://stackoverflow.com/questions/linked/23803743?lq=1 being the single most linked question about promises in SO.